js/Discourse.js

```diff
@@ -55,6 +57,97 @@ class Discourse extends React.Component {
         this._siteManager.refreshSites({ ui: false, fast: true });
       }
     };
+
+    this._handleOpenUrl = this._handleOpenUrl.bind(this);
+
+    if (Platform.OS === "ios") {
+      SafariView.addEventListener("onShow", () => {
+        this._siteManager.refreshInterval(60000);
+      });
+
+      SafariView.addEventListener("onDismiss", () => {
```

Comment:
I moved these listeners from the HomeScreen component to its parent, that way they would be triggered even when users aren't specifically in the Home screen. This might be out of scope for the PR, if so, let me know, and I'll move them back to their original place.